screens/Scan/ScanScreen.tsx

```diff
@@ -144,6 +144,51 @@ export const ScanScreen: React.FC = () => {
     );
   }
 
+  function LocalPermissionRational() {
+    return (
+      <Overlay
```

Comment:
Extract it as separate component

Reply:
created screens/Scan/LocationPermissionRational.tsx